### PR TITLE
Update TLS settings in Windows VM deployment

### PR DESCRIPTION
**Pull request by monikavar**
- update TLS to 1.2 in windows vm deployment

- PBI link is [here](https://dev.azure.com/msazure/One/_workitems/edit/12555410)

### CODE REVIEW

File: deploy/scripts/TLSSettings.ps1 [hunk 1]
```diff
@@ -0,0 +1,738 @@
...
+}
+else
+{
+
+    Write-Log -Message "You set the RebootIfRequired flag to false. If changes are made, the system will NOT reboot "  -Logfile $logLocation -Severity Information
+    Write-Log -Message "No changes will take effect until a reboot has been completed. "  -Logfile $logLocation -Severity Information
+    Write-Log -Message "Script does not include a reboot by design" -Logfile $logLocation -Severity Information
+}
+Write-Log -Message "========== End of logging for a script execution =========="  -Logfile $logLocation -Severity Information
+# SIG # Begin signature block
```
Comment (cristipogacean) [inline deploy/scripts/TLSSettings.ps1]: what is the purpose of this signature block? looks commented out to me

File: deploy/scripts/TLSSettings.ps1 [hunk 2]
```diff
@@ -0,0 +1,738 @@
+﻿#***************************************************************************************************************
```
Comment (cristipogacean) [inline deploy/scripts/TLSSettings.ps1:1]: Is this script subject to any copyright/licensing model? If so, we should add it to our thirdparty notices here: https://raw.githubusercontent.com/Azure/Industrial-IoT/main/thirdpartynotices.txt . If not, let's add our MIT header.

File: deploy/scripts/TLSSettings.ps1 (hunk 1 above)
Comment (monikavar) [reply deploy/scripts/TLSSettings.ps1]: Left by mistake. Removed

File: deploy/scripts/TLSSettings.ps1 (hunk 2 above)
Comment (monikavar) [reply deploy/scripts/TLSSettings.ps1:1]: This is not subject to any copyright/licensing model. Our scripts does not have any MIT header. Resolving the comment.

**Review by cristipogacean — APPROVED** [review]

**Pull request merged**